.clang-format

```diff
@@ -28,7 +28,7 @@ ColumnLimit: 80
 ContinuationIndentWidth: 4
 Cpp11BracedListStyle: false
 # IndentCaseBlocks: false
-IndentCaseLabels: false
+IndentCaseLabels: true
```

Comment:
:+1:

Reply:
`true` results in:

```c
   switch (planet) {
      case MERCURY:
      case VENUS:
      case EARTH:
      //...
```

`false` results in:

```c
   switch (planet) {
   case MERCURY:
   case VENUS:
   case EARTH:
   //...
```

The latter seems harder to read to me, but I don't really care too much so I'm checking only that the latter is your intention - is that the case?

Reply:
Sorry, I was trying to say that I like the change :smile: